Extension/src/LanguageServer/client.ts

```diff
@@ -745,6 +751,47 @@ class DefaultClient implements Client {
         }
     }
 
+    private promptCompileCommands(params: CompileCommandsPaths) : void {
+        if (this.configuration.Configurations[this.configuration.CurrentConfiguration].compileCommands !== undefined) {
+            return;
+        }
+
+        let showCompileCommandsSelection: PersistentState<boolean> = new PersistentState<boolean>("CPP.showCompileCommandsSelection", true);
+        if (!showCompileCommandsSelection.Value) {
+            return;
+        }
+
+        let message: string = "Would you like to use compile_command.json files to auto-configure IntelliSense?";
+        let yes: string = "Yes";
+        let notNow: string = "Not Now";
+        let dontAskAgain: string = "Don't Ask Again";
+        vscode.window.showInformationMessage(message, yes, notNow, dontAskAgain).then((value) => {
+            switch (value) {
+                case yes:
+                    if (params.paths.length > 1) {
+                        ui.showCompileCommands(params.paths).then((index) => {
+                            if (index < 0) {
+                                return;
+                            }
+                            if (index >= params.paths.length) {
```

Comment:
if we keep this, please add a comment that this is because "edit configurations" is added to the list.

Reply:
Removed after speaking with Rong :smile: